tests/aws/services/sns/test_sns_filter_policy.py

```diff
@@ -1238,34 +1238,34 @@ def test_filter_policy_ip_address_condition(
                 "source": "test-source",
                 "detail-type": "test-detail-type",
                 "account": "123456789012",
-                "region": "us-east-2",
+                "region": "my-region",
```

Comment:
Just to add some context: The `us-east-2` has been replaced with `<my-region>` automatically by the snapshot fixture due to this default configuration:
https://github.com/localstack/localstack/blob/b182caed487c050ac48d4d75c43e6ca238b00485/tests/aws/conftest.py#L114

This means that by default any values equal to the currently used region when executing the test is replaced with `<my-region>`. Since this value was static and did not change even though the region of the test changed (in our MA/MR pipeline), the value was _not_ replaced with `<my-region>` but stayed the same and therefore failed on the matching.